config/jobs/image-pushing/k8s-staging-sig-storage.sh

```diff
@@ -36,7 +36,6 @@ readonly REPOS=(
 # don't.
 readonly BROKEN_REPOS=(
     kubernetes-csi/csi-driver-iscsi
-    kubernetes-csi/csi-driver-nfs
```

Comment:
nfs repo should be added back to the REPOS list above. Then you need to run the script to update the yaml file.